assets/swagger.json

```diff
@@ -10760,6 +10760,10 @@
           "type": "string",
           "title": "Schedule is the time the window will begin, specified in cron format"
         },
+        "syncOverrun": {
+          "type": "boolean",
+          "title": "SyncOverrun allows ongoing syncs to continue in two scenarios:\nFor deny windows: allows syncs that started before the deny window became active to continue running\nFor allow windows: allows syncs that started during the allow window to continue after the window ends"
+        },
```

Comment:
This was added manually. I couldn't figure out how to generate this file automatically, `openapigen` task did not generate it